drivers/i2c/i2c_shell.c

```diff
@@ -88,82 +103,125 @@ static int cmd_i2c_recover(const struct shell *shell_ctx,
 	const struct device *dev;
 	int err;
 
-	dev = device_get_binding(argv[1]);
+	dev = device_get_binding(argv[ARGV_DEV]);
 	if (!dev) {
-		shell_error(shell_ctx, "I2C: Device driver %s not found.", argv[1]);
+		shell_error(shell_ctx, "I2C: Device driver %s not found.",
+			    argv[1]);
```

Comment:
Shouldn't this be `ARGV_DEV` instead of `1`, since this patch is getting rid of the magic number?

Reply:
I've started refactoring this, but it was merged in the meantime. Should I revert that change and create new PR with changes and improvements, or just add new PR only with improvements?